test/integration/integration_kubernetes_test.go

```diff
@@ -416,3 +416,93 @@ kubernetes:
 	// https://github.com/kubernetes/kubernetes/pull/110479
 	asserter.droppedTargetCount(t, 1)
 }
+
+func Test_CuratedExperience(t *testing.T) {
+	t.Parallel()
+
+	k8sEnv := newK8sEnvironment(t)
+
+	// Create running pod
+	podBase := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "unmonitored-pod",
+			Labels: map[string]string{
+				"k8s.io/app": "myApp",
+			},
+			Annotations: map[string]string{
+				"prometheus.io/scrape": "true",
+				"prometheus.io/scheme": "https",
+				"prometheus.io/port":   "8001",
+				"prometheus.io/path":   "/custom-path",
+			},
+		},
+		Spec: fakePodSpec(),
+	}
+
+	podNotMonitored := k8sEnv.addPodAndWaitOnPhase(t, podBase, corev1.PodRunning)
+
+	// Create a similar pod, but with the required filter
+	podBase.Name = "monitored-pod1"
+	podBase.Labels["app.kubernetes.io/name"] = "asdTesT1asd"
+
+	podMonitored1 := k8sEnv.addPodAndWaitOnPhase(t, podBase, corev1.PodRunning)
+
+	// Create a similar pod, but with a different label
+	podBase.Name = "monitored-pod2"
+	podBase.Labels["something-different2"] = "asdtest2"
+
+	podMonitored2 := k8sEnv.addPodAndWaitOnPhase(t, podBase, corev1.PodRunning)
+
+	nrConfigConfig := fmt.Sprintf(`
+newrelic_remote_write:
+  license_key: nrLicenseKey
+common:
+  scrape_interval: 1s
+kubernetes:
+  jobs:
+    - job_name_prefix: test-k8s
+      target_discovery:
+        pod: true
+        additional_config:
+         kubeconfig_file: %s
+         namespaces:
+          names:
+          - %s
+  curated_experience:
+    regexes:
+      - test1
+      - test2
+    source_labels:
+      - something-different1     
+      - app.kubernetes.io/name
+      - something-different2
```

Comment:
The new rerlabel rules would be applied to all kubenetes jobs (not the static ones).

Adding a new one would not "filter" anything since everything would be still scraped by other jobs